src/redux/rootReducer.ts

```diff
@@ -1,6 +1,8 @@
 import { combineReducers } from "redux";
+import autorizationReducer from "./reducers/autorizationReducer";
 import app from "./reducers/inputReducer";
 
 export default combineReducers({
   app,
+  autorization: autorizationReducer,
```

Comment:
you may not need this reducer at all

Reply:
You definitely do not need autorizationReducer.
Use useData as auth indicator

Reply:
fixed